### DISCUSSION TITLE
Are there any plans to link other documents as cards ?

### DISCUSSION THREAD

**Discussion by alkanna** [outline/outline#5969] [Ideas]
It would be nice to have this kind of block :

![CleanShot 2023-10-09 at 11 47 44](https://github.com/outline/outline/assets/30343764/ce7379f7-9bed-4658-9bfa-9cadd5a99eb0)

Maybe not as complex (here it's a grid), but having outline recognise its own documents and have some kind of specific block (optional ?) would look very nice instead of having raw links everywhere.

**Comment by tommoor** [comment]
Once #3000 lands we will be able to look at some alternative displays for links
